http://code.google.com/p/naev/issues/detail?id=129

opengl.c: In function ‘write_png’:
opengl.c:820:39: error: ‘Z_DEFAULT_COMPRESSION’ undeclared (first use in this function)

--- src/opengl.c
+++ src/opengl.c
@@ -39,6 +39,7 @@
 #include <string.h>
 #include <stdarg.h> /* va_list for gl_print */
 
+#include <zlib.h> /* Z_DEFAULT_COMPRESSION */
 #include <png.h>
 
 #include "SDL.h"
